ERROR - 2022-03-30 04:57:37 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: YES)
CRITICAL - 2022-03-30 04:57:37 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
#0 C:\xampp\htdocs\AttendanceSystem\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\AttendanceSystem\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\AttendanceSystem\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\AttendanceSystem\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(32): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->auth()
#6 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#7 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-03-30 04:58:37 --> Error connecting to the database: Unknown database 'attendancesystem'
CRITICAL - 2022-03-30 04:58:37 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'attendancesystem'
#0 C:\xampp\htdocs\AttendanceSystem\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\AttendanceSystem\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\AttendanceSystem\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\AttendanceSystem\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(32): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->auth()
#6 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#7 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-03-30 04:58:50 --> Error connecting to the database: Unknown database 'attendancesystem'
CRITICAL - 2022-03-30 04:58:50 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'attendancesystem'
#0 C:\xampp\htdocs\AttendanceSystem\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\AttendanceSystem\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\AttendanceSystem\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\AttendanceSystem\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(32): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->auth()
#6 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#7 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-03-30 05:00:12 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2022-03-30 05:00:12 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
#0 C:\xampp\htdocs\AttendanceSystem\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\AttendanceSystem\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\AttendanceSystem\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\AttendanceSystem\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(32): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->auth()
#6 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#7 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-03-30 05:00:27 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2022-03-30 05:00:27 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
#0 C:\xampp\htdocs\AttendanceSystem\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\AttendanceSystem\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\AttendanceSystem\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\AttendanceSystem\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(32): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->auth()
#6 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#7 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-03-30 05:01:07 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-03-30 05:01:07 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\AttendanceSystem\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\AttendanceSystem\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\AttendanceSystem\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\AttendanceSystem\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(32): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->auth()
#6 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#7 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-03-30 05:04:14 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2022-03-30 05:04:14 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
#0 C:\xampp\htdocs\AttendanceSystem\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\AttendanceSystem\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\AttendanceSystem\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\AttendanceSystem\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(32): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->auth()
#6 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#7 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-03-30 05:12:12 --> Attempt to read property "password" on array
#0 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\xampp\\htdocs...', 37)
#1 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->auth()
#2 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#3 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-03-30 05:20:20 --> syntax error, unexpected token "else"
#0 C:\xampp\htdocs\AttendanceSystem\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\AttendanceSystem\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(797): class_exists('\\App\\Controller...', true)
#4 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(406): CodeIgniter\CodeIgniter->startController()
#5 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-30 05:24:34 --> Undefined array key "phone_no"
#0 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(66): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 66)
#1 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(42): App\Controllers\Admin\AdminController->setUserSession(Array)
#2 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->auth()
#3 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#4 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-03-30 05:37:44 --> Call to undefined method App\Models\UserModel::count
#0 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(80): CodeIgniter\Model->__call('count', Array)
#1 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->dashboard()
#2 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#3 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-03-30 05:42:39 --> Call to undefined method App\Models\UserModel::list
#0 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(80): CodeIgniter\Model->__call('list', Array)
#1 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->dashboard()
#2 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#3 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-03-30 05:45:11 --> Call to undefined method App\Models\UserModel::getResult
#0 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(80): CodeIgniter\Model->__call('getResult', Array)
#1 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->dashboard()
#2 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#3 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-03-30 05:45:13 --> Call to undefined method App\Models\UserModel::getResult
#0 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(80): CodeIgniter\Model->__call('getResult', Array)
#1 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->dashboard()
#2 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#3 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-03-30 05:45:14 --> Call to undefined method App\Models\UserModel::getResult
#0 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(80): CodeIgniter\Model->__call('getResult', Array)
#1 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->dashboard()
#2 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#3 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-03-30 05:45:14 --> Call to undefined method App\Models\UserModel::getResult
#0 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(80): CodeIgniter\Model->__call('getResult', Array)
#1 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->dashboard()
#2 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#3 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-03-30 06:37:12 --> CodeIgniter\Router\RouteCollection::group(): Argument #1 ($name) must be of type string, array given, called in C:\xampp\htdocs\AttendanceSystem\app\Config\Routes.php on line 47
#0 C:\xampp\htdocs\AttendanceSystem\app\Config\Routes.php(47): CodeIgniter\Router\RouteCollection->group(Array, Object(Closure))
#1 C:\xampp\htdocs\AttendanceSystem\system\Router\RouteCollection.php(607): CodeIgniter\CodeIgniter->Config\{closure}(Object(CodeIgniter\Router\RouteCollection))
#2 C:\xampp\htdocs\AttendanceSystem\app\Config\Routes.php(48): CodeIgniter\Router\RouteCollection->group('admin', Array, Object(Closure))
#3 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(713): require('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#5 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-30 06:38:08 --> CodeIgniter\Router\RouteCollection::group(): Argument #1 ($name) must be of type string, array given, called in C:\xampp\htdocs\AttendanceSystem\app\Config\Routes.php on line 47
#0 C:\xampp\htdocs\AttendanceSystem\app\Config\Routes.php(47): CodeIgniter\Router\RouteCollection->group(Array, Object(Closure))
#1 C:\xampp\htdocs\AttendanceSystem\system\Router\RouteCollection.php(607): CodeIgniter\CodeIgniter->Config\{closure}(Object(CodeIgniter\Router\RouteCollection))
#2 C:\xampp\htdocs\AttendanceSystem\app\Config\Routes.php(48): CodeIgniter\Router\RouteCollection->group('admin', Array, Object(Closure))
#3 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(713): require('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#5 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-30 06:38:26 --> CodeIgniter\Router\RouteCollection::group(): Argument #1 ($name) must be of type string, array given, called in C:\xampp\htdocs\AttendanceSystem\app\Config\Routes.php on line 47
#0 C:\xampp\htdocs\AttendanceSystem\app\Config\Routes.php(47): CodeIgniter\Router\RouteCollection->group(Array, Object(Closure))
#1 C:\xampp\htdocs\AttendanceSystem\system\Router\RouteCollection.php(607): CodeIgniter\CodeIgniter->Config\{closure}(Object(CodeIgniter\Router\RouteCollection))
#2 C:\xampp\htdocs\AttendanceSystem\app\Config\Routes.php(48): CodeIgniter\Router\RouteCollection->group('admin', Array, Object(Closure))
#3 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(713): require('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#5 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-30 06:39:07 --> CodeIgniter\Router\RouteCollection::group(): Argument #1 ($name) must be of type string, array given, called in C:\xampp\htdocs\AttendanceSystem\app\Config\Routes.php on line 47
#0 C:\xampp\htdocs\AttendanceSystem\app\Config\Routes.php(47): CodeIgniter\Router\RouteCollection->group(Array, Object(Closure))
#1 C:\xampp\htdocs\AttendanceSystem\system\Router\RouteCollection.php(607): CodeIgniter\CodeIgniter->Config\{closure}(Object(CodeIgniter\Router\RouteCollection))
#2 C:\xampp\htdocs\AttendanceSystem\app\Config\Routes.php(48): CodeIgniter\Router\RouteCollection->group('admin', Array, Object(Closure))
#3 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(713): require('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#5 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-30 06:49:48 -->  route cannot be found while reverse-routing.
#0 C:\xampp\htdocs\AttendanceSystem\system\HTTP\RedirectResponse.php(56): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(45): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->auth()
#3 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#4 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-03-30 06:50:24 -->  route cannot be found while reverse-routing.
#0 C:\xampp\htdocs\AttendanceSystem\system\HTTP\RedirectResponse.php(56): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(45): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->auth()
#3 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#4 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-03-30 06:52:14 -->  route cannot be found while reverse-routing.
#0 C:\xampp\htdocs\AttendanceSystem\system\HTTP\RedirectResponse.php(56): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 C:\xampp\htdocs\AttendanceSystem\app\Controllers\Admin\AdminController.php(47): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(832): App\Controllers\Admin\AdminController->auth()
#3 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\AdminController))
#4 C:\xampp\htdocs\AttendanceSystem\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\AttendanceSystem\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
